src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj

```diff
@@ -27,9 +27,9 @@
     <Compile Include="System\ComponentModel\Win32Exception.Windows.cs" />
   </ItemGroup>
   <ItemGroup Condition=" '$(OS)' == 'Unix' ">
-    <Compile Include="$(CommonPath)\Interop\Interop.strerror.Unix.cs" />
-    <Compile Include="Interop\Interop.Unix.cs" />
     <Compile Include="System\ComponentModel\Win32Exception.Unix.cs" />
+    <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs"><Link>Interop\Interop.Libraries.cs</Link></Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libc\Interop.strerror.cs"><Link>Interop\Interop.strerror.cs</Link></Compile>
```

Comment:
For the the Link I've been suggesting that people use Common in the front so it easy to see these come from common. So something like Common\Interop\Interop.Libaries.cs